CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 ## HEAD (Unreleased)
 
+## 3.5.0 (June 30, 2021)
+
 - Update pulumi dependencies v3.5.1 (https://github.com/pulumi/pulumi-kubernetes/pull/1623)
```

Comment:
Since we are updating to 3.6.0 anyway
```suggestion
```

Reply:
I'd rather leave the link since the other PR also included SDK changes.